tests/spec_const/spec_const_multiple_via_kb_fp64.cpp

```diff
@@ -10,17 +10,17 @@
 #include "../common/common.h"
 #include "../common/type_coverage.h"
 
-#include "specialization_constants_multiple.h"
+#include "spec_const_multiple.h"
 
-#define TEST_NAME specialization_constants_multiple_via_kb_fp64
+#define TEST_NAME spec_const_multiple_via_kb_fp64
 
 namespace TEST_NAMESPACE {
 using namespace sycl_cts;
 
 /** test specialization constants for double
  */
 class TEST_NAME : public sycl_cts::util::test_base {
-public:
+ public:
```

Comment:
I suggest we keep existing formatting and move this change to a separate PR, unless adding 1-space indentation fixes some warnings on Windows which will be surprising (😮).

Reply:
I'm fine either way, but this formatting comes from the format file: `AccessModifierOffset: -1` which pulls it in one character as opposed to regular indentation in the scope, which would be two. I've come to set my IDE to format changes only, but it is mildly annoying that formatting isn't universal. (I do understand the drawback of losing history just because enforcing formatting in a retrospecitve manner.)